lib/utils/requireModule.js

```diff
@@ -1,25 +1,20 @@
 const path = require('path');
 
-const requireModule = module.exports = function(fullpath) {
+const requireModule = module.exports = function (fullpath) {
   let exported;
   try {
     exported = require(fullpath);
   } catch (err) {
-    if (err.code === 'MODULE_NOT_FOUND') {
-      return requireModule(path.resolve(fullpath));
```

Comment:
Did you check whether require takes care of this or not?

Reply:
`require()` works with `__dirname` which is independent of what the current directory was when our app was loaded. It is always the directory where the module is located, so it is consistent.

Reply:
@beatfactor I wanted to understand why we added this `if` condition, because `require(fullPath)` will always resolve to an absolute path. I had removed it because I think we don't need to try again. Let me know your opinion